packages/design-system-tokens/package.json

```diff
@@ -3,11 +3,6 @@
   "version": "0.0.1",
   "private": true,
   "description": "CMSDS Design System Tokens",
-  "devDependencies": {
```

Comment:
These devDeps exist in the root package.json. Removing from this file had no obvious impact on token scripts.

Reply:
I actually think that if it's used by a script _in this package_ it should stay a dependency of this package even if it's a duplicate. Downside to that is we need to try to keep the version numbers in sync.